k8s/charts/seaweedfs/values.yaml

```diff
@@ -570,12 +570,19 @@ filer:
     # Suffix of the host name, {bucket}.{domainName}
     domainName: ""
     # enable user & permission to s3 (need to inject to all services)
-    enableAuth: false
+    enableAuth: true
     skipAuthSecretCreation: false
     # set to the name of an existing kubernetes Secret with the s3 json config file
     # should have a secret key called seaweedfs_s3_config with an inline json configure
-    existingConfigSecret: ""
+    existingConfigSecret: false
```

Comment:
This doesn't sound like a Boolean.

Reply:
You are correct, it should be a string. I will alter my approach accordingly 👍